index.js

```diff
@@ -22,4 +22,38 @@ addTaskButton.addEventListener("click", () => {
   });
 
   console.log(tasks);
+
+  taskTable = document.getElementById("task-table");
+  const TaskDetails = document.createElement("tr");
```

Comment:
Why not camel case?

Reply:
I remember you said it's better to have same type for naming id's ,don't you?

Reply:
I thought it's better to have same type of naming for id's,so I use dash and lowerCase as always.
But if I'm wrong please tell me

Reply:
I mean `TaskDetails`.

Reply:
That was my mistake , I fixed it